### PR TITLE
Resolving Issue#915

### PR DESCRIPTION
**Pull request by j0rt3g4**
Fixing some formatting on the text and the answer is provided on the article.

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Solving issue #915

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 1]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
 
 ## See Also
 
-[Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.<br>
-[What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.<br>
-[What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.<br>
+[Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
- [Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.  
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 2]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
 
 ## See Also
 
-[Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.<br>
-[What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.<br>
-[What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.<br>
+[Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.  
+
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 3]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
+[What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
- [What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 4]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
+
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 5]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
+[What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
- [What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 6]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
+
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
```

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 7]
```diff
@@ -231,8 +228,12 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
 [Windows 10 Fall Creators Update Next Generation Security](https://www.youtube.com/watch?v=JDGMNFwyUg8): YouTube video about Windows Defender ATP in Windows 10, version 1709.
```
Comment (nenonix) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
- [Windows 10 Fall Creators Update Next Generation Security](https://www.youtube.com/watch?v=JDGMNFwyUg8): YouTube video about Windows Defender ATP in Windows 10, version 1709.
```

**Comment by mypil** [comment]
@greg-lindsay  - Can you please assist to check, approve and merge the content of this PR based on issue #915?

Thank you.

File: windows/whats-new/whats-new-windows-10-version-1803.md [hunk 8]
```diff
@@ -231,8 +228,9 @@ Support in [Windows Defender Application Guard](#windows-defender-application-gu
...
 
-[Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.<br>
-[What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.<br>
-[What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.<br>
-[Windows 10 Fall Creators Update Next Generation Security](https://www.youtube.com/watch?v=JDGMNFwyUg8): YouTube video about Windows Defender ATP in Windows 10, version 1709.
+- [Windows 10 Features](https://www.microsoft.com/windows/features): Review general information about Windows 10 features.  
+- [What's New in Windows 10](https://docs.microsoft.com/windows/whats-new/): See what’s new in other versions of Windows 10.
+- [What's new in Windows 10, version 1709](https://docs.microsoft.com/windows-hardware/get-started/what-s-new-in-windows): See what’s new in Windows 10 hardware.
+- [Windows 10 Fall Creators Update Next Generation Security](https://www.youtube.com/watch?v=JDGMNFwyUg8): YouTube video about Windows Defender ATP in Windows 10, version 1709.
+
```
Comment (illfated) [inline windows/whats-new/whats-new-windows-10-version-1803.md]:
```suggestion
```
(there is a blank line below already)

**Comment by mypil** [comment]
@greg-lindsay - We would like to follow up. Can you please assist to check, approve and merge the content of this PR based on issue #915?

Thank you.

**Comment by illfated** [comment]
(Ref. obsolete PR #3709 )

**Pull request merged**